moveZeroes/index.js

```diff
@@ -15,4 +15,22 @@ var moveZeroes = function (nums) {
         i++;
     }
     return nums;
+};
+
+
+// another solution move zeroes
+
+var moveZeroes = function(nums) {
+    let count = 0
+   nums.forEach((_, index)=> {
+       if (nums[index] !== 0){
+           nums[count] = nums[index]
+           count++
+       } })
+      for (let i = count; i < nums.length; i ++){
```

Comment:
should we not be starting from count - 1
If the array's actual length was 5, by the end of it would be 6